### ISSUE TITLE
Disabled numbers when used in the 9 grids

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#11]
```
In pop and sn modes, disable numbers being used 9 times in the grid,
correctly placed or not. it could be a simple setting.
```

Original issue reported on code.google.com by `magic.cr...@gmail.com` on 14 Jul 2009 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
could be a nice option
```

Original comment by `cicci...@gmail.com` on 27 Jul 2009 at 12:19

**Comment by GoogleCodeExporter** [comment]
```
 agreed, almost posted my own request, such a setting is probably easy to implement and also is 
featured in some of the sudoku apps already on the market, however I prefer 
this one, its by far the 
best ;)
```

Original comment by `rik...@gmail.com` on 29 Aug 2009 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
This cannot be easily done in 0.6.x due to some problems in its architecture. 

It's easy in current unstable version (0.7.0). I'll release some test version 
on this
site in 2 weeks, however I'm not able to tell when this will be stable enough 
to go
on android market.
```

Original comment by `romario...@gmail.com` on 31 Aug 2009 at 3:43
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
if you need any testers, it IS open source, I'll send my feedback :)

I'm not COMPLETELY stupid when it comes to linux/android ;)
```

Original comment by `rik...@gmail.com` on 2 Sep 2009 at 2:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 7 Sep 2009 at 5:35
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Solved in 0.7.0. You can wait for this release or download test version here:
http://code.google.com/p/opensudoku-android/downloads/list (look for OpenSudoku 
0.7.0
- unstable).
```

Original comment by `romario...@gmail.com` on 21 Sep 2009 at 5:25
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 1 Oct 2009 at 9:05
- Changed state: **Verified**
